src/modules/series/SeriesSlice.ts

```diff
@@ -42,24 +41,7 @@ const getComputePlanSeries = async (
             { signal }
         );
         cpStats = response.data.compute_plan_statistics;
-    } catch (error) {
```

Comment:
I really don't know much about React, but from what I gather, there used to be 2 `try / catch`  blocks, and now there is only one. 

In Python at least, it is usually considered a good practice to have exception blocks as small as possible; I don't know if that would apply here or not 🤷

Reply:
Yes, there were 2 exceptions blocks, as there were two requests made (namely `ComputePlansApi.listComputePlanPerformances(...)` and `getAllPages(...)`), as we have now only one request, there is only one catch block for the error of this request (if you look at it, previous L45-L51 is exactly the same than previous L63 to 69)

Reply:
Oh, my bad, the api call is actually higher up, Github presentation induced me in error.